wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@JonWel , personally I would make a local variable like that:

local_buffer=(s*repeat("\0",buffer_length)).data

That way, you have buffer of bigger size, zero-terminated, local. I hope that Julia has scope concept?